uber/templates/jobs/index.html

```diff
@@ -1,9 +1,27 @@
 {% extends "base-admin.html" %}
 {% block title %}Shifts{% endblock %}
+{% block page_styles %}
+<!--inside page_style -->
+<link rel="stylesheet" href="../static/styles/fullcalendar.min.css" />
+<link rel="stylesheet" href="../static/styles/fullcalendar.print.css" media="print"/>
```

Comment:
Yeah I think in this case we don't need to worry about adding it to our bulk files, since the plugin is used one only one page (and we have to do it this way for some plugins that aren't available on NPM). Consolidating these plugins is honestly a huge pain right now because you have to install NPM, then use NPM to install bower and grunt AND grunt-bower-concat with the -g (global) flag, THEN symlink your nodejs app to the thing bower expects (something which I literally have to google every single time), _THEN_ actually get to the process of adding the plugin as a requirement in bower.json and minifying it. This is like a 20-minute process every time I try to do it and it drives me crazy, so... let's not throw you under that bus for this PR. :)